modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -53,13 +54,20 @@ const setupEvents = (editor: Editor) => {
     }
   };
 
-  const scroll = (e) => Events.fireScrollContent(editor, e);
+  const scroll = (e: EventUtilsEvent<Event>) => Events.fireScrollContent(editor, e);
 
   DOM.bind(contentWindow, 'resize', resizeWindow);
   DOM.bind(contentWindow, 'scroll', scroll);
 
   // Bind to async load events and trigger a content resize event if the size has changed
-  const elementLoad = DomEvent.capture(Element.fromDom(editor.getBody()), 'load', resizeDocument);
+  const elementLoad = DomEvent.capture(Element.fromDom(editor.getBody()), 'load', (e) => resizeDocument(e.raw));
+
+  editor.on('hide', () => {
+    Css.set(uiComponents.uiMothership.element, 'display', 'none');
```

Comment:
Maybe pull this out into a separate variable? Otherwise, this won't minify very well.

Also, I could be wrong here, but isn't element a function here? (that's changed in the develop branch since we removed the thunks)